### PR TITLE
new post: 6 tips for your next (remote) onboarding

### PR DESCRIPTION
**Pull request by fejes713**
A shot, list-style blog post on _6 tips for your next (remote) onboarding_

### CODE REVIEW

**Comment by fejes713** [comment]
also @Chalarangelo is there a reason for duck being named _ducke_?

**Comment by Chalarangelo** [comment]
> also @Chalarangelo is there a reason for duck being named _ducke_?

Haha, yes I found it very amusing to name him/it _ducke_ like a mix between Duck and Duke :laughing:

File: blog_posts/6-tips-for-your-next-(remote)-onboarding.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+type: list
+tags: career
+authors: fejes713
+cover: blog_images/golden-gate-bridge.jpg
+excerpt: Many developers find onboarding experience worrisome. Here are 6 helpful tips to make your onboarding experience a pleasant one.
+---
+
+Many developers find onboarding experience worrisome. Here are 6 helpful tips to make your onboarding experience a pleasant one:
+
+1. Ask for a project-specific tech stack and look for stuff that you might not be familiar with. One month is enough to cover the basics of anything you didn't learn at University and will save you lots of time (and stress) once the internship starts.
```
Comment (Chalarangelo) [inline blog_posts/6-tips-for-your-next-(remote)-onboarding.md]: We could generalize this just a little more, like this:
```suggestion
1. Ask for a project-specific tech stack and look for stuff that you might not be familiar with. One month is enough to cover the basics of anything you didn't learn at University or have no prior experience on and will save you lots of time (and stress) once your job or internship starts.
```
What do you think?

File: blog_posts/6-tips-for-your-next-(remote)-onboarding.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+2. Don't be embarrassed about asking for help. To get where they are now, your teammates had to go through the onboarding process and stuff you're facing now. Define with your team when you should ask for help. You'll run into obstacles every day. Some might take 30 minutes to solve, and others might take forever. Find a balance - You don't want to interrupt the whole team every 30 minutes, but don't let one problem turn your internship into an unpleasant experience.
```
Comment (Chalarangelo) [inline blog_posts/6-tips-for-your-next-(remote)-onboarding.md]:
```suggestion
2. Don't be embarrassed about asking for help. To get where they are now, your teammates had to go through the onboarding process and all the stuff you're facing now. Define when you should ask for help with your team. You'll run into obstacles every day. Some might take 30 minutes to solve, and others might take forever. Find a balance - You don't want to interrupt the whole team every 30 minutes, but don't let one problem turn your job/internship into an unpleasant experience.
```

File: blog_posts/6-tips-for-your-next-(remote)-onboarding.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+3. Most likely, you won't be micromanaged, but it's a good idea to keep everybody updated on your progress. Doing this, your team might offer some useful advice based on your progress or prepare you for upcoming obstacles in the task. If something isn't right, don't be silent. Your teammates aren't wizards, and they can't read your mind. If something is bothering you, you're stuck or not comfortable doing something. Just talk. Everybody is willing to help, but you need to let them know first.
```
Comment (Chalarangelo) [inline blog_posts/6-tips-for-your-next-(remote)-onboarding.md]:
```suggestion
3. Most likely, you won't be micromanaged, but it's a good idea to keep everybody updated on your progress. Doing this, your team might offer some useful advice based on your progress or prepare you for upcoming obstacles in the task. If something isn't right, don't be silent. Your teammates aren't wizards, and they can't read your mind. If something is bothering you, you're stuck or not comfortable doing something - Just talk! Everybody is willing to help, but you need to let them know first.
```
Very minor suggestions, but I think we should emphasize the `Just talk` part as it is in the middle of the poin and could be missed more easily, exclamation mark to the rescue.

File: blog_posts/6-tips-for-your-next-(remote)-onboarding.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+
+![Duck icon](./blog_images/ducke.png)
+
+4. Practice [rubber duck debugging](https://rubberduckdebugging.com/). You don't want to schedule a 30-minute meeting with your team and 2 minutes into explaining the problem you get to the _aha!_ moment where you figure out the solution to the problem. In a nutshell, before asking for help, try to explain the problem you're having to a random object. Most of the time, during this process, you'll get new ideas and solve a problem by yourself without needing somebody's help.
+
+5. If time allows it, plan virtual pair-programming sessions with your teammates. Once a week, pick a short task and try to solve it together. Doing this, you will pick up domain-specific knowledge faster, learn how to navigate new codebase efficiently, and improve your problem-solving skills.
+
+6. Socialize with other team members on Slack channels like _#random_ and _#watercooler_. Just because you're new to the team doesn't mean you shouldn't participate in casual conversations with other employees. They are social beings too and are probably looking forward to getting to know better new colleagues.
```
Comment (Chalarangelo) [inline blog_posts/6-tips-for-your-next-(remote)-onboarding.md]: Our company watercooler slack channel is called, `#blahblah`, idk if that's common. Should we add it here or not? Up to you.

File: blog_posts/6-tips-for-your-next-(remote)-onboarding.md [hunk 5]
```diff
@@ -0,0 +1,26 @@
+---
+title: 6 tips for your next (remote) onboarding
+type: list
+tags: career
```
Comment (Chalarangelo) [inline blog_posts/6-tips-for-your-next-(remote)-onboarding.md]: I also used `programming` and `jobs` in a similar post about careers, I think we can add them here for ease of searching on the website.

**Review by Chalarangelo — COMMENTED** [review]
Great job! I've suggested a few very minor changes and we should be good to go!

File: blog_posts/6-tips-for-your-next-(remote)-onboarding.md (hunk 4 above)
Comment (fejes713) [reply blog_posts/6-tips-for-your-next-(remote)-onboarding.md]: Hahaha I think there're many ways to call it. #watercooler is the most common I think

Comment (Chalarangelo) [reply blog_posts/6-tips-for-your-next-(remote)-onboarding.md]: Yep, it gets the points across as-is. No reason to clutter it.

**Review by Chalarangelo — APPROVED** [review]
:tada: Great job, merging right away!

**Pull request merged**